src/types.ts

```diff
@@ -496,10 +496,7 @@ export interface GetSwapQuoteRequestParams {
     gasPrice?: BigNumber;
     excludedSources?: ERC20BridgeSource[];
     affiliateAddress?: string;
-    rfqt?: {
-        intentOnFilling?: boolean;
-        isIndicative?: boolean;
-    };
+    rfqt?: Partial<RfqtRequestOpts>;
```

Comment:
I think we only want this to be `intentOnFilling`, `isIndicative` and `nativeExclusivelyRFQT`, not the other `RfqtRequestOpts` -- is that correct?

If so, use `Pick` instead of `Partial`

Reply:
`Pick< RfqtRequestOpts , 'intentOnFilling ' ....>`

Reply:
done